src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int bothside;
```

Comment:
The variable name 'bothside' is not descriptive and may cause confusion. It is better to use 'width' or 'base' for a rectangle to clearly represent the dimensions.